pom.xml

```diff
@@ -185,19 +185,6 @@
 						</execution>
 					</executions>
 				</plugin>
-				<plugin>
-					<groupId>org.eclipse.tycho</groupId>
-					<artifactId>tycho-surefire-plugin</artifactId>
-					<version>${tycho-version}</version>
-					<configuration>
-						<useUIHarness>${tycho.surefire.useUIHarness}</useUIHarness>
-						<useUIThread>${tycho.surefire.useUIThread}</useUIThread>
-						<includes>
-							<include>**/*Test.java</include>
```

Comment:
Looks like with the default Pattern we have now an extra test, but it is failing and has to be fixed.